### PR TITLE
Swap aggregator styling + copy updates

### PR DESCRIPTION
**Pull request by l2succes**
Fixes TEAM2-114 
Fixes TEAM2-102 
Fixes TEAM2-104

## What changed (plus any additional context for devs)

- Updated search input styling
- Updated copy on swap explainer modal
- Updated "Search" -> "Search tokens"

## Dev checklist for QA: what to test

- Swap token search input

## Final checklist

- [ ] Assigned individual reviewers?
- [ ] Added labels?
- [ ] Added e2e tests? if not please specify why
- [ ] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Review by estebanmino — COMMENTED** [review]
lint is failing 😢

File: src/components/exchange/ExchangeSearch.js [hunk 1]
```diff
@@ -138,6 +157,9 @@ const ExchangeSearch = (
   const { isSearchModeEnabled = true } = useContext(DiscoverSheetContext) || {};
 
   const spinnerTimeout = useRef();
+  const { colors } = useTheme();
+  const shadows = useMemo(() => DiscoverButtonShadowsFactory(colors), [colors]);
```
Comment (estebanmino) [inline src/components/exchange/ExchangeSearch.js:161]: wondering if we really need `useMemo` here

File: src/components/exchange/ExchangeSearch.js [hunk 2]
```diff
@@ -107,6 +121,11 @@ const timingConfig = {
   duration: 300,
 };
 
+const DiscoverButtonShadowsFactory = colors => [
```
Comment (skylarbarrera) [inline src/components/exchange/ExchangeSearch.js:124]: can we rename this and increase the shadows a lil bit? looking a lil light I think

**Comment by l2succes** [comment]
@skylarbarrera Updated the shadow 
@estebanmino fixed the linter errors

**Review by skylarbarrera — APPROVED** [review]
👨‍🍳 😘

**Pull request merged**